### PR TITLE
[doc] Update postgres-npgsql.md

### PR DESCRIPTION
**Pull request by jrnker**
Rephrased text to emphasize the recommended connection behaviour to avoid loading types on open.  Added connection string for this recommendation.

### CODE REVIEW

**Review by ddorian — APPROVED** [review]
LGTM

File: docs/content/stable/drivers-orms/csharp/postgres-npgsql.md [hunk 1]
```diff
@@ -121,12 +121,20 @@ Copy the following code to the `Program.cs` file to set up YugbyteDB tables and
...
-On every new connection, the Npgsql driver also makes [extra system table queries to map types](https://github.com/npgsql/npgsql/issues/1486), which adds significant overhead. To turn off this behavior, set the following option in your connection string builder:
+On every new connection, the Npgsql driver also makes [extra system table queries to map types](https://github.com/npgsql/npgsql/issues/1486), which adds significant overhead. It is recommended to turn this off to significantly reduce connection open execution time.
+
+Set the following option in your connection string builder:
 
 ```csharp
 connStringBuilder.ServerCompatibilityMode = ServerCompatibilityMode.NoTypeLoading;
 ```
 
+Alternatively you can add the following to your connection string:
```
Comment (aishwarya24) [inline docs/content/stable/drivers-orms/csharp/postgres-npgsql.md:132]:
```suggestion
Alternatively, you can add the following to your connection string:
```

File: docs/content/stable/drivers-orms/csharp/postgres-npgsql.md [hunk 2]
```diff
@@ -121,12 +121,20 @@ Copy the following code to the `Program.cs` file to set up YugbyteDB tables and
 
 {{< warning title="Warning" >}}
 
-On every new connection, the Npgsql driver also makes [extra system table queries to map types](https://github.com/npgsql/npgsql/issues/1486), which adds significant overhead. To turn off this behavior, set the following option in your connection string builder:
+On every new connection, the Npgsql driver also makes [extra system table queries to map types](https://github.com/npgsql/npgsql/issues/1486), which adds significant overhead. It is recommended to turn this off to significantly reduce connection open execution time.
```
Comment (aishwarya24) [inline docs/content/stable/drivers-orms/csharp/postgres-npgsql.md:124]:
```suggestion
On every new connection, the Npgsql driver also makes [extra system table queries to map types](https://github.com/npgsql/npgsql/issues/1486), which adds significant overhead. It is recommended to turn off this behaviour to significantly reduce connection open execution time.
```

**Review by aishwarya24 — COMMENTED** [review]
Some minor comments. Thanks @jrnker !

**Review by Sfurti-yb — APPROVED** [review]
LGTM

**Pull request merged**